frontend/src/ui/pages/MyBankAccountsPage.js

```diff
@@ -34,6 +34,28 @@ class MyBankAccountsPage extends Component {
     }
   }
 
+  async removeBankAccount(bankAccount) {
+    this.setState({ loading: true })
+    try {
+      // Default estimation of gas is too low, multiply it by 2
+      const gasEstimate = await this.pobaContract.unregisterBankAccount.estimateGas(
+        bankAccount.account,
+        bankAccount.bankName,
+        { from: this.state.ethAccount }
+      )
+      await this.pobaContract.unregisterBankAccount(bankAccount.account, bankAccount.bankName, {
+        from: this.state.ethAccount,
+        gas: gasEstimate * 2
```

Comment:
The transaction failed consistently because the estimated gas was too low. Tested different values and arrived to the min calculation that makes the transaction succeed.